### PR TITLE
@broskoski: add related artists rail

### PR DESCRIPTION
**Pull request by xtina-starr**
https://github.com/artsy/force/issues/74

related artists rail

### CODE REVIEW

File: apps/artwork/components/related_artists/artists.jade [hunk 1]
```diff
@@ -0,0 +1,21 @@
+each artist in relatedArtists
```
Comment (broskoski) [inline apps/artwork/components/related_artists/artists.jade]: Not necessarily something that needs addressing now, but this should probably be made into a component now that this template is in two places.

File: apps/artwork/components/related_artists/index.styl [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    h2
+      garamond-size('headline', true)
+
+  .hoverable-image-link
+    width 100%
+
+  .artwork-artist-rail-cell
+    mgr-n-up(4, 30px, (3/4), false)
+
+.artwork-artist-related-rail__view-all
```
Comment (broskoski) [inline apps/artwork/components/related_artists/index.styl]: You can give this one the same style as the above:
```
&__view-all
   // etc
```

**Comment by broskoski** [comment]
Nice specs! Looks good to me, the above comments are pretty minor, feel free to merge when ready.

**Comment by xtina-starr** [comment]
Okay cool, I'll go ahead and merge. I come back later this week to refactor this into it's own component.

**Pull request merged**